### DISCUSSION TITLE
AssertionError: This version of Atoti only supports atoti-jupyterlab3 v0.8.6 but got v0.8.3.

### DISCUSSION THREAD

**Discussion by LeighTilley** [atoti/atoti#813] [Q&A] [answered]
Hey

I just updated Atoti and I see 0.8.6. I ran Jupyter Lab and I see error:

'AssertionError: This version of Atoti only supports atoti-jupyterlab3 v0.8.6 but got v0.8.3.'

I just tried to install atoti-jupyterlab3 with --ignore-installed and I still see 0.8.3 in pip freeze for this package. I also just uninstalled atoti-jupyterlab3 and then did an install with --no-cache-dir and it downloaded 0.8.3.

![image](https://github.com/atoti/atoti/assets/9533038/38ec322f-cfdb-4391-b67e-c7abe3047378)

How do I get atoti-jupyterlab3 0.8.6 please?

Thanks

Leigh

**Comment by tibdex** [comment, marked as answer]
Hi Leigh, you're right the `assert` is wrong. It will be fixed in the next release (next month). In the meantime, you have a few options:
- Replace `assert entry_point_name == "jupyterlab"` with `assert entry_point_name == "jupyterlab3"` in the file raising the `AssertionError` (`atoti_core.get_installed_plugins`)
- Disable `assert` statements by running your script with `python -O`
- Update to `atoti-jupyterlab`

**Comment by LeighTilley** [comment]
Hi

OK thanks for your prompt reply. I'm quite new to the Atoti stuff, as in day job I was immersed in the Java side of it all :),  so just looking around; where do I find the files to edit please?

thanks

Leigh

https://tilleytech.com

**Reply by tibdex** [reply]
It's kinda hacky since this requires editing a dependency file. You should see the file path in the full traceback of the `AssertionError`.

**Reply by LeighTilley** [reply]
Hi

OK thanks got it working cheers. ;)

Leigh
https://tilleytech.com

**Reply by tibdex** [reply]
This is fixed in 0.8.7.

**Reply by LeighTilley** [reply]
Hey hey

OK thanks mate; appreciate the follow-up too!

I shall update it! :)

Thanks alot

Leigh
https://tilleytech.com
